python/pfs/drp/stella/measureCrosstalk.py

```diff
@@ -0,0 +1,345 @@
+
+
+class MeasureCrosstalkConfig(Config):
+    """Configuration for MeasureCrosstalkTask"""
+    isr = ConfigurableField(target=IsrTask, doc="Instrument signature removal")
+    subtractBackground = ConfigurableField(target=SubtractBackgroundTask, doc="Subtract background")
+    buildFiberTraces = ConfigurableField(target=BuildFiberTracesTask, doc="Build fiber traces")
+    traceRadius = Field(dtype=float, default=30.0, doc="Half-width of trace to exclude from target")
+    doRerunIsr = Field(dtype=bool, default=True, doc="Rerun the ISR, even if postISRCCD files are available")
+    badMask = ListField(dtype=str, default=["SAT", "BAD", "INTRP"], doc="Mask planes to ignore")
+    equationThreshold = Field(dtype=float, default=0.05,
+                              doc="Threshold on ratio of eigenvalue to sum of eigenvalues for solving matrix")
+    imageThreshold = Field(dtype=float, default=50.0, doc="Threshold on trace-masked image to toss CRs")
+
+    def setDefaults(self):
+        Config.setDefaults(self)
+        self.isr.doOverscan = True
+        self.isr.doBias = True
+        self.isr.doDark = True
+        self.isr.doFlat = False
+        self.isr.doWrite = False
+        self.isr.doLinearize = False
+        self.isr.doBrighterFatter = False
+        self.isr.doAddDistortionModel = False
+        self.isr.doCrosstalk = False
+        self.isr.doCrosstalkBeforeAssemble = False
+        self.isr.overscanFitType = "AKIMA_SPLINE"
+        self.isr.overscanOrder = 30
+        # Turn off all kinds of interpolation: we don't want the images corrected
+        self.isr.growSaturationFootprintSize = 0  # We want the saturation spillover: it's good signal
+        self.isr.doDefect = False
+        self.isr.doWidenSaturationTrails = False
+        self.isr.doSaturationInterpolation = False
+        self.isr.maskListToInterpolate = []
+        # Background subtraction
+        self.subtractBackground.statisticsProperty = "MEDIAN"
+        self.subtractBackground.binSize = 512
+        self.subtractBackground.useApprox = False
+        MaskX.addMaskPlane("FIBERTRACE")
+        self.subtractBackground.ignoredPixelMask.append("FIBERTRACE")
+        self.buildFiberTraces.doBlindFind = True  # Traces shift around in a dithered flat sequence
+
+
+class MeasureCrosstalkRunner(TaskRunner):
+    def __call__(self, *args, **kwargs):
+        """Remove the dataRef from the result
+
+        Otherwise, running threaded yields:
+
+            sqlite3.ProgrammingError: SQLite objects created in a thread can
+            only be used in that same thread.
+        """
+        result = TaskRunner.__call__(self, *args, **kwargs)
+        if hasattr(result, "dataRef"):
+            del result.dataRef
+        return result
+
+
+class MeasureCrosstalkTask(CmdLineTask):
+    """Measure intra-CCD crosstalk
+
+    This Task behaves in a scatter-gather fashion:
+    * Scatter: get ratios for each CCD.
+    * Gather: combine ratios to produce crosstalk coefficients.
```

Comment:
Shouldn't it mention here (or in the ``reduce`` method) that the cross-talk coefficients are also written to file?